### DISCUSSION TITLE
curl_mime_data_cb man page: curl_off_t datasize undocumented

### DISCUSSION THREAD

**Discussion by JustZhenya** [curl/curl#7479] [Q&A]
Hello!

I want to use `curl_mime_data_cb` in my application, what should I set `datasize` to when I don't know the size of my data? I'm sending the output of ffmpeg through pipe, so I can't predict target size.

**Comment by bagder** [comment]
@monnerat you recall?

**Comment by monnerat** [comment]
(curl_off_t) -1, AFAICR
